src/relay-runtime/src/__tests__/RelayLazyLogger.test.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * @flow
+ * @jest-environment jsdom
```

Comment:
Isn't this already what we use in this workspace?

Reply:
It is, however, we should also have a Node.js test which is currently missing. You are the author actually in the eager logger and I like to keep it there explicitly. 😊

Reply:
Sure, makes sense 👌